action.yml

```diff
@@ -17,78 +17,75 @@ inputs:
   max-tool-memory:
     required: false
-    description: "Maximum of allowed memory for each tool execution"
+    description: "Maximum allowed memory for each tool (bytes)"
   max-allowed-issues:
     required: false
     default: "2147483647"
     description: "Maximum number of issues allowed for the analysis to succeed"
   tool:
     required: false
-    description: "The name of the tool to analyze the code"
+    description: >-
+      Only run a specific tool or tool category (metrics, issues, duplication).
+      For a full list, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
   tool-timeout:
     required: false
-    description: "Maximum time each tool has to execute"
+    description: "Tool execution timeout (e.g. 15minutes, 1hour)"
   upload:
     required: false
     description: "Upload analysis results to Codacy"
   upload-batch-size:
     required: false
-    description: "Maximum number of results in each batch to upload to Codacy"
+    description: "Maximum number of uploaded results per batch"
   fail-if-incomplete:
     required: false
     description: "Fail the analysis if any tool fails to run"
   allow-network:
     required: false
-    description: "Allow network access for tools"
+    description: "Allow the tools to access the network"
   force-file-permissions:
     required: false
     description: "Force files to be readable by changing the permissions before running the analysis"
   gh-code-scanning-compat:
     required: false
     description: >-
-      Reduce issue severity by one level, for non-security issues, for compatibility with GitHub's code scanning feature.
-      This option will only have an effect when used in conjunction with '--format sarif'.
+      Reduce issue severity by one level for non-security issues, for compatibility with GitHub's code scanning feature.
+      This option only has an effect when used with 'format: sarif'.
   run-docker-tools:
     required: false
     default: "true"
     description: >-
-      Run dockerized tools. Possible values (true|false)
-      Check the full list in https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
+      Run all dockerized tools supported by Codacy.
+      For a full list, see https://docs.codacy.com/repositories-configure/codacy-configuration-file/#which-tools-can-be-configured-and-which-name-should-i-use
   run-gosec:
     required: false
-    description: >-
-      Run GoSec. Possible values (true|false)
+    description: "Run GoSec"
   run-staticcheck:
     required: false
-    description: >-
-      Run StaticCheck. Possible values (true|false)
+    description: "Run StaticCheck"
   clang-tidy-output:
     required: false
-    description: >-
-      Path to a file containing the output of Clang Tidy.
+    description: "Path to a file to save the output of Clang Tidy"
```

Comment:
This is actually the file containing the output of Clang-Tidy (that the users must run before the GitHub Action).

The GitHub Action then parses this output, converts it into a JSON understood by Codacy, and uploads it using our API, [see more details here](https://github.com/codacy/codacy-clang-tidy#sending-the-results-to-codacy).

We should still fix the tool name to include the "-". :nerd_face:

Reply:
Thanks for this tip!